config/config.exs

```diff
@@ -0,0 +1,6 @@
+import Config
+
+config :screens_config,
+  config_s3_bucket: "mbta-ctd-config"
+
+import_config "#{Mix.env()}.exs"
```

Comment:
Nit: I don't know if this really makes a difference but I know that there's a preference for using `config_env()` over `Mix.env()` starting at Elixir 1.9 when `Mix.Config` was deprecated. Mentioned at the bottom of [this section of the docs](https://hexdocs.pm/elixir/1.12/Config.html#module-migrating-from-use-mix-config)